material/admin/static/material/admin/css/base.css

```diff
@@ -459,3 +459,20 @@ ul.side-nav.module-menu li.active>a>span.badge {
         padding: 30px 50px 50px 50px;
     }
 }
+
+/*
+Bug fixes
+ */
+
+/* Add Scroll to Navbar */
+.side-nav {
+    overflow:scroll !important;
```

Comment:
why not fix here? https://github.com/viewflow/django-material/blob/master/material/admin/static/material/admin/css/base.css#L102